src/components/NewMessageDetail.vue

```diff
@@ -132,24 +132,7 @@ export default {
 					}
 				}
 			} else {
-				// New or mailto: message
-				logger.debug('composing a new message or handling a mailto link', {
-					threadId: this.$route.params.threadId,
-				})
-
-				let accountId
-				// Only preselect an account when we're not in a unified mailbox
-				if (this.$route.params.accountId !== 0 && this.$route.params.accountId !== '0') {
-					accountId = parseInt(this.$route.params.accountId, 10)
-				}
-
-				return {
-					accountId,
-					to: this.stringToRecipients(this.$route.query.to),
-					cc: this.stringToRecipients(this.$route.query.cc),
-					subject: this.$route.query.subject || '',
-					body: this.$route.query.body ? detect(this.$route.query.body) : html(''),
-				}
+				throw new Error('new message details can only be for replies and forwards')
```

Comment:
`Error` conflicts with the imported error component

what you want to throw is a new instance of https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error

throwing a new constructor instance causes an error

> [Vue warn]: Error in render: "TypeError: _Error__WEBPACK_IMPORTED_MODULE_7__.default is not a constructor"

Import the error component under a different name. then the global javascript `Error` will be unhidden and work as expected.

You can trigger the error if you open `/apps/mail/box/priority/thread/new`